riva/clients/nmt/streaming_s2s_client.cc

```diff
@@ -318,6 +318,10 @@ StreamingS2SClient::ReceiveResponses(std::shared_ptr<S2SClientCall> call, bool a
   std::vector<int16_t> pcm_buffer;
   std::vector<unsigned char> opus_buffer;
   while (call->streamer->Read(&call->response)) {  // Returns false when no more to read.
+    if (!call->response.speech().audio().length()) {
+      // If the audio size is zero exit from the loop.
+      break;
```

Comment:
in case when there are multiple sentences in the output stream, we can't break out at end of first sentence. So, need to use continue here instead of break.

Reply:
thanks for pointing that out.